packages/mdc-icon-button/_mixins.scss

```diff
@@ -140,3 +142,19 @@
     }
   }
 }
+
+///
+/// Sets the density scale for icon button.
+///
+/// @param {Number | String} $density-scale - Density scale value for component.
+///     Supported density scale values range from `-6` to `0`, with `0` being the default.
+///
+@mixin mdc-icon-button-density($density-scale, $query: mdc-feature-all()) {
+  $height: mdc-density-prop-value(
+    $density-config: $mdc-icon-button-density-config,
+    $density-scale: $density-scale,
+    $property-name: height,
+  );
+
+  @include mdc-icon-button-size($height / 2, $query: $query);
```

Comment:
I think we would still need to clean up this mixin. This mixin needs to be deprecated since it provides too much of low level customization including padding and different sizes for height & width instead of single dimension.

I'm thinking we would rename this to something else and migrate existing clients to that mixin internally.

Every component has a size or height mixin which acts as a proxy to density mixin, it needs to be consistent with other components.

WDYT?

Reply:
I agree, we should keep consistency and update the components. I fixed it, let me know what you think.